source/re/index.rst

```diff
@@ -102,6 +102,18 @@ Afin de rechercher une expression, on utilisera la fonction :py:func:`~re.search
     searchObj.group() :  Cats are smarter than dogs
     searchObj.group(1) :  Cats
     searchObj.group(2) :  smarter
+	
+re.match() vs re.search()
+-------------------------
+
+La fonction :py:func:`~re.match()` va vérifier une correspondance seulement au début de la chaîne, tandis que :py:func:`~re.search()` va vérifier une correspondance sur toute la chaîne.
+
+.. code:: pycon
+
+    >>> import re
+    >>> re.match("c", "abcdef") # Pas de correspondance
+    >>> re.search("c", "abcdef") # Une correspondance
+    <_sre.SRE_Match object; span=(2, 3), match='c'>
```

Comment:
J'aime bien le `is None` utilisé par Armin. http://lucumr.pocoo.org/2015/11/18/pythons-hidden-re-gems/